packages/@aws-cdk-containers/ecs-service-extensions/lib/extensions/open-telemetry.ts

```diff
@@ -0,0 +1,243 @@
+import * as ecs from '@aws-cdk/aws-ecs';
+import * as iam from '@aws-cdk/aws-iam';
+import * as ssm from '@aws-cdk/aws-ssm';
+import { Service } from '../service';
+import { ContainerMutatingHook, ServiceExtension } from './extension-interfaces';
+import { Construct } from '@aws-cdk/core';
+
+const AWS_OPEN_TELEMETRY = 'amazon/aws-otel-collector:latest';
+
+interface OpenTelemetryConfigProps {
+  serviceName: string,
+}
+
+// This helper function generates an open telemetry config
```

Comment:
Curious if this is the default ecs config, and if it should live in a separate file for easier editing?